ds3-autogen-cli/src/main/java/com/spectralogic/autogen/cli/Arguments.java

```diff
@@ -2,18 +2,29 @@
 
 
 public class Arguments {
-    private final String targetDir;
-    private final GeneratorType type;
-    private final String inputSpec;
-    private final boolean help;
+    private String targetDir;
+    private GeneratorType type;
+    private String inputSpec;
+    private boolean help;
 
-    public Arguments(final String targetDir, final GeneratorType type, final String inputSpec, final boolean help) {
```

Comment:
I think the old final stuff would have worked as is.  We only needed to check for the help command before validating the other inputs.